src/backend/api/views.py

```diff
@@ -1,3 +1,24 @@
-# from rest_framework.response import Response
-# from rest_framework import status
-# from rest_framework.decorators import api_view
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Neighborhood
+from .serializers import NeighborhoodBoundsSerializer, NeighborhoodSerializer
+
+
+class ListNeighborhoods(APIView):
+    permission_classes = (IsAuthenticated,)
```

Comment:
There's nothing technically private about this data (and indeed, the existing data is publicly available in this repo) but I didn't want someone being able to access the API without signing up for an account in case someone does something strange and potentially problematic like writing a bot to scrape the data.